packages/cli/src/parse-args.ts

```diff
@@ -260,13 +267,15 @@ const commands: ICommand[] = [
       pr,
       context,
       { ...deleteFlag, description: 'Delete old comment' },
+      { ...editFlag, description: 'Edit old comment' },
```

Comment:
Since the `--edit` flag is only used for the `comment` command can you move the full `editFlag` def here?

Similar to: https://github.com/intuit/auto/blob/ff1d187cb994e25f61655fb4ce8b5d93c47d2414/packages/cli/src/parse-args.ts#L303

Reply:
@hipstersmoothie I moved the editFlag as you suggested.

For the linting errors, I fixed them all except the one in auto.ts. It's complaining about "no-collapsible-if  if statement can be merged with previous else" on lines 435 and 439.
But I'm unable to refactor it in a way such that the linter is happy and the code is still readable
Do you have any suggestions for this?